### PR TITLE
port api.graph to rust-proto-new

### PR DESCRIPTION
**Pull request by jgrillo-grapl**
### Which issue does this PR correspond to?

https://app.zenhub.com/workspaces/grapl-6036cbd36bacff000ef314f2/issues/grapl-security/issue-tracker/907

### What changes does this PR make to Grapl? Why?

This PR ports the contents of the protobuf package `graplinc.grapl.api.graph.v1beta1` and the `rust-proto::graph_descriptions` to `rust-proto-new`.

### How were these changes tested?
Automated tests

### CODE REVIEW

File: src/rust/rust-proto-new/tests/test_utils/strategies.rs [hunk 1]
```diff
@@ -120,6 +120,396 @@ pub mod pipeline {
...
+        }
+    }
+
+    //
+    // Property
+    //
+
+    pub fn properties() -> impl Strategy<Value = Property> {
+        prop_oneof![
+            decrement_only_int_props().prop_map(|p| Property::DecrementOnlyIntProp(p)),
```
Comment (colin-grapl) [inline src/rust/rust-proto-new/tests/test_utils/strategies.rs:350]: FWIW you can do `.prop_map(Property::DecrementOnlyIntProp)`
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=fd8d5f3fe7a1a595cffc8563e1a6508c

**Review by colin-grapl — APPROVED** [review]

File: src/rust/rust-proto-new/tests/test_utils/strategies.rs (hunk 1 above)
Comment (jgrillo-grapl) [reply src/rust/rust-proto-new/tests/test_utils/strategies.rs]: done

File: src/rust/rust-proto-new/tests/proto_serde_tests.rs [hunk 2]
```diff
@@ -36,65 +36,220 @@ proptest! {
...
-        publish_raw_log_response in strategies::pipeline_ingress::publish_raw_log_responses()
-    ) {
-        check_encode_decode_invariant(publish_raw_log_response)
+mod pipeline {
+    use strategies::pipeline as st;
+
+    use super::*;
+
+    proptest! {
+        #[test]
```
Comment (wimax-grapl) [inline src/rust/rust-proto-new/tests/proto_serde_tests.rs:188]: thanks for doing this, i got lazy lol

File: src/rust/rust-proto-new/tests/test_utils/strategies.rs [hunk 3]
```diff
@@ -120,6 +120,396 @@ pub mod pipeline {
...
+        }
+    }
+
+    //
+    // Strategy
+    //
+
+    pub fn strategies() -> impl Strategy<Value = GraphStrategy> {
+        prop_oneof![
+            sessions().prop_map(|session| GraphStrategy::Session(session)),
```
Comment (wimax-grapl) [inline src/rust/rust-proto-new/tests/test_utils/strategies.rs:327]: .prop_map(GraphStrategy::Session) etc etc

**Review by wimax-grapl — APPROVED** [review]

File: src/rust/rust-proto-new/tests/test_utils/strategies.rs (hunk 3 above)
Comment (jgrillo-grapl) [reply src/rust/rust-proto-new/tests/test_utils/strategies.rs]: done

**Pull request merged**